src/ecdsa/keys.py

```diff
@@ -614,6 +614,18 @@ def to_der(
             der.encode_bitstring(point_str, 0),
         )
 
+    def to_ssh(self):
```

Comment:
the public format normally includes a comment too, shouldn't we have that as an optional function argument?

Reply:
That can be added afterwards if really needed. From other implementations it does not look to be something it is being used:

https://github.com/pyca/cryptography/blob/main/src/cryptography/hazmat/primitives/serialization/ssh.py#L745

Reply:
just an idea; I'm not insisting on it